lxqt-admin-user/mainwindow.cpp

```diff
@@ -76,6 +78,17 @@ void MainWindow::reloadUsers()
     }
 }
 
+void MainWindow::onTabChange(int index)
```

Comment:
Guard the whole function definition with the `#if`. There is no need to have it at all for non-Q_OS_FREEBSD

And for all preprocessor conditionals, please, do not indent them (hash should be the first character in the line).